### PR TITLE
[fix](partial-update) insert only without auto_inc column should not use partial update

### PR DESCRIPTION
**Pull request by cambyzju**
For primary key tables and unique_key_partial_update enabled:
1. for `insert into table values(xxx)`, we should try full column insert instead of partial update;
2. If `insert into table(cols)` include all columns except the AUTO_INCREMENT column, we should use full column insert;

### CODE REVIEW

**Comment by cambyzju** [comment]
run buildall

**Comment by cambyzju** [comment]
run buildall

**Comment by cambyzju** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by bobhan1 — APPROVED** [review]

File: regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+                qt_select_1 """ select name from test_primary_key_partial_update_auto_inc order by name; """
+
+                sql """ set enable_unique_key_partial_update=true; """
+                sql """ insert into test_primary_key_partial_update_auto_inc values(100,"doris3"); """
+                sql """ set enable_unique_key_partial_update=false; """
+                sql """ insert into test_primary_key_partial_update_auto_inc values(101, "doris4"); """
+                sql "sync"
+                qt_select_2 """ select name from test_primary_key_partial_update_auto_inc order by name; """
+
```
[the remark this answers is not in the document]
Comment (zhannngchen) [reply regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy:60]: +1

**Review by zhannngchen — CHANGES_REQUESTED** [review]
Is there any case for this?
> for insert into table values(xxx), we should try full column insert instead of partial update;

**Comment by cambyzju** [comment]
this one:
`insert into test_primary_key_partial_update_auto_inc values(100,"doris3");`

> Is there any case for this?
> 
> > for insert into table values(xxx), we should try full column insert instead of partial update;

File: regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy (hunk 1 above)
[the remark this answers is not in the document]
Comment (cambyzju) [reply regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy:60]: got, I will update it later

File: regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                            PROPERTIES("replication_num" = "1", "enable_unique_key_merge_on_write" = "${use_mow}"); """
+
+                sql """ set enable_unique_key_partial_update=true; """
+                sql """ insert into test_primary_key_partial_update_auto_inc(name) values("doris1"); """
+                sql """ set enable_unique_key_partial_update=false; """
+                sql """ insert into test_primary_key_partial_update_auto_inc(name) values("doris2"); """
+                sql "sync"
+
+                qt_select_1 """ select name from test_primary_key_partial_update_auto_inc order by name; """
+
```
[the remark this answers is not in the document]
Comment (cambyzju) [reply regression-test/suites/unique_with_mow_p0/partial_update/test_partial_update_auto_inc.groovy:54]: done

**Comment by cambyzju** [comment]
run buildall

**Comment by cambyzju** [comment]
run buildall

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**